TODO Api/app/utils/exceptions.py

```diff
@@ -0,0 +1,35 @@
+from abc import abstractmethod, ABC
+
+
+class RaiseError(Exception, ABC):
+    def get_message_for_dev(self):
+        pass
+
+    def get_message_for_client(self):
+        pass
+
+
+class InvalidIDError(Exception, ABC):
+    def __init__(self, invalid_id: int, item_type: str):
+        self.invalid_song_id = invalid_id
+        self.item_type = item_type
+        super().__init__()
+
+    def get_message_for_dev(self):
```

Comment:
Avoid writing method names like this by using conjunction & prepositions. This could be simply written as `dev_error` or `dev_messag`